pom.xml

```diff
@@ -25,14 +25,14 @@
 
     <groupId>com.connexta.search</groupId>
     <artifactId>query-api</artifactId>
-    <version>0.3.1-SNAPSHOT</version>
+    <version>0.4.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
     <properties>
         <root.package>com.connexta.search</root.package>
 
         <!-- Build properties -->
-        <java.version>11</java.version>
+        <java.version>1.8</java.version>
```

Comment:
Continuing convo on new PR https://github.com/connexta/ion-query-api/pull/36#discussion_r352814523

Reply:
@lessarderic, yes, building the api with java 8 works for both ion-search and the ion-source. ion-search can use the java 8 dependencies without any changes.